plugins/dialog/guest-js/index.ts

```diff
@@ -77,6 +77,72 @@ interface SaveDialogOptions {
   canCreateDirectories?: boolean
 }
 
+/**
+ * Default buttons for a message dialog.
+ *
+ * @since 2.3.0
+ */
+export type MessageDialogDefaultButtons =
```

Comment:
Should we use kebab case (`ok-cancel`) or camel case (`okCancel`) for these?

Reply:
Same as above, PascalCase seems better here